spec/models/tenant_spec.rb

```diff
@@ -79,6 +79,23 @@
     end
   end
 
+  describe "#display_type" do
+    before do
+      tenant  = FactoryGirl.create(:tenant, :parent => default_tenant)
+      project = FactoryGirl.create(:tenant, :parent => default_tenant, :divisible => false)
```

Comment:
```ruby
let(:tenant) { FactoryGirl.build(:tenant, :parent => default_tenant, :divisible => true) }
let(:project) { FactoryGirl.create(:tenant, :parent => default_tenant, :divisible => false) }
```

1. Use a `let` blocks for this.
2. Using `build` will make quicker
3. For these, explicitly state `:divisible` since that is what you are testing. (personal preference)